src/main/java/dukeclasses/Ui.java

```diff
@@ -0,0 +1,135 @@
+package dukeclasses;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
```

Comment:
Perhaps add a blank link between io and util imports, to be consistent with the other files